dragonchain_sdk/dragonchain_client.py

```diff
@@ -1300,3 +1422,41 @@ def _build_bitcoin_transaction_body(
         body["change"] = change_address
 
     return body
+
+
+def _build_binance_transaction_body(amount: int, to_address: str, symbol: Optional[str] = "BNB", memo: Optional[str] = "") -> Dict[str, Any]:
+    """Build the json (dictionary) body for a binance transaction given its inputs
+
+    Args:
+        amount (int): amount of token in transaction
+        to_address (str): hex of the address to send to
+        symbol (str, optional): the exchange symbol for the token (default is BNB)
+        memo (str, optional): string of data to publish in the transaction
+
+    Raises:
+        TypeError: with bad parameter types
+
+    Returns:
+        Dictionary body to use for sending a binance transaction
+    """
+
+    if symbol is not None and not isinstance(symbol, str):
+        raise TypeError('Parameter "symbol" must be of type str.')
+    if amount is not None and not isinstance(amount, int):
+        raise TypeError('Parameter "amount" must be of type int.')
+    if to_address is not None and not isinstance(to_address, str):
+        raise TypeError('Parameter "to_address" must be of type str.')
+    if memo is not None and not isinstance(memo, str):
+        raise TypeError('Parameter "memo" must be of type str.')
+
+    body = cast(Dict[str, Any], {"version": "1", "symbol": symbol, "amount": amount, "to_address": to_address, "memo": memo})
```

Comment:
I'm confused by this. This means that your body will always have all the keys with `null` (python `None`) in any field that doesn't exist.

I think you meant for this line to be `body = cast(Dict[str, Any], {"version: "1", "amount": amount, "to_address": to_address})` since those are all the required parameters. Then simply optionally add in the symbol or memo below if (and only if) they exist

Reply:
Good catch -- yes, that is what I meant to do.  Fixed!